NewBank/newbank/server/NewBank.java

```diff
@@ -52,4 +57,40 @@ private String showMyAccounts(CustomerID customer) {
 		return (customers.get(customer.getKey())).accountsToString();
 	}
 
+	private String move(CustomerID customer, String[] splitRequest){
```

Comment:
Do you think that this function would better fit inside the Customer class? So we could leave this function to parse the string only, but move the whole logic related to accounts elsewhere

Reply:
It could probably return boolean, and based on that we could send FAIL or SUCCESS to the client from here. In this way, we unbind the bank logic from the communication protocol logic